validation/dam_break_2d/plot_dam_break_results.jl

```diff
@@ -55,20 +66,23 @@ function plot_results(axs, ax_max, files)
                       normalization_factor_pressure
         pressure_P2 = json_data["pressure_P2_fluid_1"]["values"] ./
                       normalization_factor_pressure
+
+        label_prefix = occursin("reference", json_file) ? "Reference" : ""
+
         lines!(axs[1], time, pressure_P1,
-               label="dp=$(convert_to_float(split(replace(basename(json_file), ".json" => ""), "_")[end]))",
+               label="$label_prefix dp=$(convert_to_float(json_file))",
```

Comment:
```suggestion
               label="$(label_prefix)dp=$(convert_to_float(json_file))",
```
Otherwise, you're adding a space in front of the non-reference labels.

Reply:
Also, has to be `extract_resolution_from_filename` now.

Reply:
That's probably why the tests are failing.

Reply:
Not resolved?